src/shared/components/mutationMapper/MutationMapperStore.ts

```diff
@@ -78,9 +78,11 @@ export default class MutationMapperStore
                     return this.getMutations();
                 }
             } else {
-                if (this.activeTranscript === undefined) {
-                    this.activeTranscript = canonicalTranscriptId;
-                }
+                try {
+                    if (this.activeTranscript === undefined) {
+                        this.activeTranscript = canonicalTranscriptId;
```

Comment:
@inodb this code (setting instance prop inside of getter) is an antipattern and quite possibly a bug.  when this method is accessed inside a computed property or observer component (which does in fact happen), mobx throws an error in order to avoid endless loop.   we should looking into this

Reply:
@alisman 	i see - thanks! Will revisit when fixing this https://github.com/cBioPortal/cbioportal/issues/3522